skyplane/replicate/replicator_client.py

```diff
@@ -256,12 +256,13 @@ def provision_gateway_instance(region: str) -> Server:
         # Firewall rules
         # todo add firewall rules for Azure
         public_ips = [self.bound_nodes[n].public_ip() for n in self.topology.gateway_nodes]
+        private_ips = [self.bound_nodes[n].private_ip() for n in self.topology.gateway_nodes if n.region.split(":")[0] == "gcp"]
         authorize_ip_jobs = []
         authorize_ip_jobs.extend(
             [partial(self.aws.add_ips_to_security_group, r.split(":")[1], public_ips) for r in set(aws_regions_to_provision)]
         )
         if gcp_regions_to_provision:
-            authorize_ip_jobs.append(partial(self.gcp.add_ips_to_firewall, public_ips))
+            authorize_ip_jobs.append(partial(self.gcp.add_ips_to_firewall, public_ips + private_ips))
```

Comment:
Do we have to specifically authorize private IPs for GCP to GCP transfers?

Reply:
I was testing GCP to GCP transfer without authorizing private IPs but it seems to get stuck

Reply:
OK sounds good, we should keep it then